taxonomies/additives.txt

```diff
@@ -6313,7 +6313,7 @@ wikidata:en:Q420130
 wikipedia:en:https://en.wikipedia.org/wiki/Sodium_propionate
 
 
-en:E282, calcium propionate, calcium propanoate
+en:E282, calcium propionate, calcium propanoate, cal. pro., cal.pro.
```

Comment:
"cal. pro." with dot + space will not work:
https://world.openfoodfacts.org/cgi/test_ingredients_analysis.pl?ingredients_text=+cal.+pro.%2C+cal.pro.&type=add&action=process&submit=Submit+Query

Reply:
That's right, it does not work. I will leave it in the taxonomy anyway, as it could work in the future if we change the parsing algorithm.